apps/studio/src/features/editing-experience/components/EditLinkPreview.tsx

```diff
@@ -0,0 +1,80 @@
+import type { IsomerSitemap } from "@opengovsg/isomer-components"
+import { useAtomValue } from "jotai"
+
+import { useQueryParse } from "~/hooks/useQueryParse"
+import { editLinkSchema } from "~/pages/sites/[siteId]/links/[linkId]"
+import { trpc } from "~/utils/trpc"
+import { linkAtom } from "../atoms"
+import PreviewWithoutSitemap from "./PreviewWithoutSitemap"
+import { ViewportContainer } from "./ViewportContainer"
+
+export const EditLinkPreview = (): JSX.Element => {
```

Comment:
nit: can rename this to `EditCollectionLinkPreview`, otherwise can be confusing with normal link editing

Reply:
sounds good